spec/index.html

```diff
@@ -65,22 +82,98 @@
       are comprised of a default graph and zero or more named graphs.</li>
   </ul>
 
-  <p>RDF 1.2 Concepts also introduces key concepts and terminology, and discusses
-    datatyping and the handling of <a>fragment identifiers</a> in IRIs within
+  <p>RDF 1.2 introduces <a>quoted triples</a> as a fourth kind of <a>RDF term</a>
+    which can be used as the <a>subject</a> or <a>object</a> of another <a>triple</a>,
+    making it possible to make statements about other statements.</p>
+
+  <p>RDF 1.2 Concepts introduces key concepts and terminology for RDF 1.2, discusses
+    datatyping, and the handling of <a>fragment identifiers</a> in IRIs within
     RDF graphs.</p>
 </section>
 
 <section id="sotd">
   <p>This document is part of the RDF 1.2 document suite.
     It is the central RDF 1.2 specification and defines the core RDF concepts.
     Test suites and implementation reports of a number of RDF 1.2
-    specifications that build on this document are available through the  
+    specifications that build on this document are available through the
     [[[RDF11-TESTCASES]]] document [[RDF11-TESTCASES]].
     There have been no changes to this document since its publication as
     Proposed Recommendation.</p>
 
   <p>RDF 1.2 Concepts is an update to [[RDF11-CONCEPTS]],
     which was itself, an update to [[RDF-CONCEPTS-20040210]].</p>
+
+  <div class="ednote">
+    <p>This draft introduces placeholder terminology
+      adapted from [[?RDF-STAR-CG]].
+      These terms are tentative and subject to change or removal
+      in a future version of this specification:</p>
+
+    <dl class="sort">
+      <dt><a>quoted triple</a></dt>
+      <dd>Originally defined as <a data-cite="?RDF-STAR-CG#dfn-quoted">quoted triple</a>
+        in [[?RDF-STAR-CG]] as an <a data-cite="?RDF-STAR-CG#dfn-triple">RDF-star triple</a>
+        which is the subject or object of another
+        <a data-cite="?RDF-STAR-CG#dfn-triple">RDF-star triple</a>.
+        Alternate names include <em>triple term</em>.</dd>
+      <dt><a>asserted triple</a></dt>
+      <dd>Originally defined as <a data-cite="?RDF-STAR-CG#dfn-asserted">asserted triple</a>
+        in [[?RDF-STAR-CG]] as an <a data-cite="?RDF-STAR-CG#dfn-triple">RDF-star triple</a>
+        which is an element of an <a data-cite="?RDF-STAR-CG#dfn-graph">RDF-star graph</a>.</dd>
+      <dt><a>complex triple</a></dt>
```

Comment:
"complex*" "plain*" are not in the CG report.

Nor is compliance.
Profile is, referring to [Content Negotiation by Profile](https://www.w3.org/TR/dx-prof-conneg/#introduction).

Reply:
Whether it's "profile" of "conformance" is still being considered in https://github.com/w3c/rdf-star-wg/issues/19. Content Negotiation would be outside the scope of RDF Concepts. I think the point is that some implementations may not support quoted triples, and there should probably be a way to discuss this. We can either re-visit this when the conversation is more complete, or remove the language altogether until there is something more substantial.